server/subsonic/stream.go

```diff
@@ -56,7 +56,13 @@ func (c *StreamController) Stream(w http.ResponseWriter, r *http.Request) (*resp
 	} else {
 		// If the stream doesn't provide a size (i.e. is not seekable), we can't support ranges/content-length
 		w.Header().Set("Accept-Ranges", "none")
-		w.Header().Set("Content-Type", stream.ContentType())
+		contentType := stream.ContentType()
+		switch contentType {
+		case "audio/mp4":
```

Comment:
If we are hard coding this here, why not just set the mime type in the `consts/mime_types.go` file?

Anyway, is it safe to do this? What happens in other browsers?